### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by TrySound**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,6 +1,32 @@
...
+  "version": "1.2.0",
+  "homepage": "http://jakiestfu.github.io/Ripple.js/demo",
+  "authors": [
+    "Jacob Kelley <jakie8@gmail.com>"
+  ],
+  "description": "Adds Material style ripple to anything",
+  "main": [
+    "src/ripple.js",
+    "src/ripple.css"
+  ],
```
Comment (jakiestfu) [inline bower.json:11]: Even though the source files happen to be a 1-1 to uncompressed dist files, I'd still prefer to reference the files in the dist folder instead. That being said, if we ever decide to use less as a src, this approach will not work. Thoughts?

Also, we'll need to update to 1.2.1 :)

Comment (TrySound) [inline bower.json:11]: So you may do non-minified and minified in dist folder. And both with your header.

Comment (TrySound) [inline bower.json:11]: I do not use grunt (gulp lover), so don't know how to do that.

File: Gruntfile.js [hunk 2]
```diff
@@ -60,17 +60,6 @@ module.exports = function(grunt) {
...
-            name: pkg.name,
-            author: pkg.author,
-            version: pkg.version,
-            main: 'dist/ripple.min.js'
-        };
-        fs.writeFileSync('bower.json', JSON.stringify(bower, null, "\t"));
-    });
-
-    grunt.registerTask('default', ['jshint', 'uglify', 'cssmin', 'sync_versions']);
-    grunt.registerTask('develop', ['jshint', 'uglify', 'cssmin', 'sync_versions', 'watch']);
```
Comment (jakiestfu) [inline Gruntfile.js:75]: I can handle the additional grunt stuff, but why did you delete this?

Comment (TrySound) [inline Gruntfile.js:75]: Bower use auto version by github tags. So this is unnecessary. And bower must contain more info. And I add this info.

I understand grunt just know))

**Pull request merged**